README.md

```diff
@@ -1339,6 +1339,7 @@ Software that does not fit in another section.
 - [DOMJudge](https://www.domjudge.org/) - A system for running a programming contest, like the ICPC regional and world championship programming contests. ([Demo](https://www.domjudge.org/demo), [Source Code](https://github.com/DOMjudge/domjudge)) `GPL-2.0/BSD-3-Clause/MIT` `PHP`
 - [F-Droid](https://f-droid.org) - Server tools for maintaining an F-Droid repository system. ([Source Code](https://gitlab.com/fdroid/fdroidserver)) `AGPL-3.0` `Python/Docker`
 - [Flagsmith](https://flagsmith.com) - Flagsmith provides a dashboard, API and SDKs for adding Feature Flags to your applications. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD-3-Clause` `Docker`
+- [GO Feature Flag](https://gofeatureflag.org) - GO Feature Flag is a simple, complete, and lightweight feature flag solution 100% Open Source. Alternative to LaunchDarkly. ([Source Code](https://github.com/thomaspoignant/go-feature-flag)) `MIT` `Go`
```

Comment:
My personal opinion but I think 

```suggestion
- [GO Feature Flag](https://gofeatureflag.org) - GO Feature Flag is a simple, complete, and lightweight feature flag solution that is 100% Open Source. Alternative to LaunchDarkly. ([Source Code](https://github.com/thomaspoignant/go-feature-flag)) `MIT` `Go`
```
would read better, even then, I would say the 100% open source isnt needed here as you provide a link to the source code.